src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.command.Command;
+import duke.command.CommandParser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+
+/**
+ * Duke class to initialise application
```

Comment:
Should there be punctuation at the end of each line? I noticed that most of your JavaDocs do not have punctuation. 
```suggestion
 * Duke class to initialise application.
```